service/skeleton/Dockerfile

```diff
@@ -1,4 +1,19 @@
+# Container in which to build the application
+FROM adoptopenjdk/openjdk11-openj9:jdk-11.0.1.13-alpine-slim as builder
+
+# Copy the source code into the builder container
+WORKDIR /micronaut/src/@app.name@
+COPY . .
+
+# Build the application in the builder container
+RUN ./gradlew assemble
```

Comment:
This change assumes that the user is using Gradle and it will fail if they use Maven.